### DISCUSSION TITLE
Error with complex (*.yob)

### DISCUSSION THREAD

**Discussion by wonwill** [YAMACS-SML/YAMACS#7] [General]
I saw another question about the error with the ligand.
https://github.com/YAMACS-SML/YAMACS/discussions/2

And I watched your video several times. But I don't still get there.
https://www.youtube.com/watch?v=s4iqiCdN-MI

I got the complex.yob from YASARA Structure.
In my case, could you help me out how I should prepare the MD system?

**Comment by YAMACS-SML** [comment]
Please watch the video given in the following link 
https://github.com/YAMACS-SML/YAMACS/blob/main/video_tutorials/4_protein_ligand_complex_acpype.webm
Make sure that you have already installed 'acpype' and 'antechamber' (if you are using Amber force field, then you must need it to generate the .itp file of ligand). For more information regarding the installation of acpype and antechamber, please visit the following link: 
https://www.researchgate.net/post/How-to-install-ACPYPER-and-ANTEAMBER-software-in-Linux
https://pypi.org/project/acpype/

If you are using charmm36 force field. Then it will automatically generate the .itp file from https://www.swissparam.ch/ website. 

In the described video link, only upto the energy minimization steps were performed. After this step, you need to click on MD-Run plugin to initiate the simulation. 
If you face any further difficulties, feel free to contact us.
